client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -295,20 +263,41 @@ export class DomainDetailsForm extends PureComponent {
 			title = this.props.translate( 'G Suite Account Information' );
 		} else {
 			title = this.props.translate( 'Domain Contact Information' );
-			message = this.props.translate(
-				'For your convenience, we have pre-filled your WordPress.com contact information. Please ' +
-					"review this to be sure it's the correct information you want to use for this domain."
-			);
+			if ( hasDomainProduct ) {
+				if ( hasSomeDomainsWithPrivacy ) {
+					if ( hasAllDomainsWithPrivacy ) {
+						message = this.props.translate(
+							'We have pre-filled the required contact information from your WordPress.com account. Privacy ' +
+								'Protection is included to protect your identity. {{a}}Learn more{{/a}}',
```

Comment:
This should be "... is included to help protect your personal information." per @kelliepeterson's comment in the thread at p99Zz8-Ql-p2